lib/absinthe/phase/document/execution/resolution.ex

```diff
@@ -143,16 +143,13 @@ defmodule Absinthe.Phase.Document.Execution.Resolution do
     |> to_result(bp_field, full_type)
     |> walk_result(acc, bp_field, full_type, info)
   end
+  defp build_result({:error, params}, acc, bp_field, info, _)
+   when is_list(params) or is_map(params) do
+    {[message: msg], extra} = Keyword.split(Enum.to_list(params), [:message])
```

Comment:
If `:message` isn't provided, the "Resolution function did not return" error (defined as the fallthrough for `build_result/5` should be used -- and:

- [ ] That error should be updated to reflect the new possible return values
- [ ] Also, the `Absinthe.Type.Field.error_output` typespec needs to be changed